blueprints/10-l4tm_packages/blueprint.py

```diff
@@ -135,6 +156,31 @@ def _load_data():
             _data.update(dict((pkg['Package'], pkg) for pkg in tmp))
 
 
+def get_all_mirrors():
+    """
+        Return a list of sources.list entries from the tmms config parameters.
+    """
+    mirror = BP.config['DEBIAN_MIRROR']
+    release = BP.config['DEBIAN_RELEASE']
+    areas = list(BP.config['DEBIAN_AREAS'])
```

Comment:
Isn't this already a list?

Reply:
Nope. Sometimes it is a set. Sometimes it is a list - depends who modifies tmms.
Safer just to force list all the time.

Reply:
How did it get to be a set, you mean if someone uses "set()" in the /etc/tmms file?

Reply:
nope.
AREAS = ("main", "contrib", "non-free")

This will make it a set.

Reply:
No, that makes it a tuple, which is an immutable list.  The construct

AREAS = frozenset("main", "contrib", "non-free")

is what makes a set.   Now, if a tuple's immutability is a problem, then list() is the right answer.